.ko.yaml

```diff
@@ -1 +1 @@
-defaultBaseImage: registry.k8s.io/build-image/go-runner:v2.3.1-go1.19.4-bullseye.0
+defaultBaseImage: registry.k8s.io/build-image/go-runner:v2.3.1-go1.19.7-bullseye.0
```

Comment:
1.26 is on an older go version than 1.25?

Reply:
This particular patch version is. We're also discussing maybe moving away from this go-runner image since it seems to be quite far behind in availability for new golang versions. It only exists for golang 1.19 up to 1.19.7 and 1.20 up to 1.20.2 today, which are both older.

Reply:
Gotcha, makes sense.